github/authorizations.go

```diff
@@ -273,11 +142,15 @@ func (s *AuthorizationsService) Delete(ctx context.Context, id int64) (*Response
 //
 // The returned Authorization.User field will be populated.
 //
-// GitHub API docs: https://developer.github.com/v3/oauth_authorizations/#check-an-authorization
-func (s *AuthorizationsService) Check(ctx context.Context, clientID string, token string) (*Authorization, *Response, error) {
-	u := fmt.Sprintf("applications/%v/tokens/%v", clientID, token)
+// GitHub API docs: https://developer.github.com/v3/apps/oauth_applications/#check-a-token
```

Comment:
According to this URL, a custom media header is needed for this endpoint:
`application/vnd.github.doctor-strange-preview+json`

Can you please add this, and anywhere else it is needed?

There are many examples in the surrounding code for how custom media headers are added (and how they are also tested, which helps us to maintain the repo by alerting us in tests when they need to be removed).